charts/asset-manager/templates/worker-deployment.yaml

```diff
@@ -45,6 +45,20 @@ spec:
             allowPrivilegeEscalation: false
           # TODO: consider implementing a liveness probe for Sidekiq
           # e.g. https://github.com/arturictus/sidekiq_alive
+        - name: clamdb
+          image: "{{ required "A valid .Values.appImage.repository entry required!" .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
+          command: ["/bin/sh"]
+          args: ["-c", "freshclam", "-d"]
```

Comment:
I would put freshclam -d in a single parameter, i.e. args: ["-c", "freshclam -d"]

Ref: https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#run-a-command-in-a-shell

Reply:
i guess both approaches will have equal affect https://linux.die.net/man/1/bash. i will update thanks

Reply:
updated